SETUP.md

```diff
@@ -38,6 +38,12 @@ You can do Code.org development using OSX, Ubuntu, or Windows (running Ubuntu in
 
 1. `bundle exec rake install`
     * This can take a long time, ~30 minutes or more. The most expensive are the "seeding" tasks, where your local DB is populated from data in the repository. Some of the seeding rake tasks can take several minutes. The longest one, `seed:scripts`, can take > 10 minutes, but it should at least print out progress as it goes.
+
+1. fix your database charset and collation to match our servers
+    * `bin/dashboard-sql`
+    * `ALTER DATABASE dashboard_development CHARACTER SET utf8 COLLATE utf8_unicode_ci;`
+    * `ALTER DATABASE dashboard_test CHARACTER SET utf8 COLLATE utf8_unicode_ci;`
```

Comment:
The long term solution here is a bit uncertain, but the consequences of not doing this during setup right now are that you risk ending up with diffs in schema.rb which show up after each time you do `rake build` or `rake db:migrate`. Much inconclusive discussion [here](https://codedotorg.slack.com/archives/C0T10HG6N/p1601513126001300) and [here](https://codedotorg.slack.com/archives/C0T0PNTM3/p1603225836458900).

Reply:
@jamescodeorg @sureshc @wjordan this is the workaround I'm proposing in the short term to deal with character set and collation issues on developer machines.